### ISSUE TITLE
Do not show 'Approve' tokens screen when ETH option is selected/tokens are approved

### ISSUE THREAD

**Issue by elena-zh** [gnosis/cowswap#2168]
When a user selects an investment 'Buy vCOW with ETH' option only, I think, it would be nice to skip the next screen 'Claiming vCOW is a two step process', as approval ETH token is not needed

See the video: https://watch.screencastify.com/v/T5Cf88lgLHuJ4fxtWyYF

The same is for the case when a user has already approved USDC or GNO token: 
1. 'Buy vCOW with GNO' option
2. Approve GNO
3. Go back to the 'vCOW claim breakdown' screen
4. then return to the 'Set allowance to Buy vCOW' screen: so here we could skip  'Claiming vCOW is a two step process' before step 4 screen

**Comment by anxolin** [comment]
Good point, although we can classify as low prio

**Comment by elena-zh** [comment]
No longer relevant after #2246 implementation
